src/main/java/mate/jdbc/dao/DriverDao.java

```diff
@@ -0,0 +1,16 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Driver;
+
+public interface DriverDao {
+    Driver create(Driver driver);
+
+    Driver get(Long id);
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-JDBC/dao-vs-service/dao-vs-service_checklist.html#:~:text=find()%20methods%20in%20Dao%20layer%20should%20return%20Optional%20(not%20in%20services).%20In%20the%20service%20layer%20better%20call%20method%20get()%20on%20Optional%20and%20return%20the%20object.